docs/03-usage.md

```diff
@@ -46,27 +43,43 @@ Remember to re-run Snowpack every time you import an new dependency.
+| `--no-source-map`   | `boolean`  | `false`                                            | Skip emitting source map files (`.js.map`) into `dest`.                                                                                                                                     |
+| `--remote-package`  | `string[]` |                                                    | `name,version` pair(s) of packages that should be left unbundled and referenced remotely. Example: `foo,v4` will rewrite all imports of `foo` to `{remoteUrl}/foo/v4` (see `--remote-url`). |
+| `--remote-url`      | `string`   | `https://cdn.pika.dev`                             | Configures the domain where remote imports point to.                                                                                                                                        |
+| `--nomodule`        | `string`   |                                                    | Your app’s entry file for generating a `<script nomodule>` bundle                                                                                                                           |
+| `--nomodule-output` | `string`   | `app.nomodule.js`                                  | Filename for nomodule output                                                                                                                                                                |
+
+You can see a list of all these options with the `--help` flag without having to refer back to these docs:
 
 ```bash
 npx snowpack --help
 ```
 
+### Configuration
 
-### All Config Options
+All CLI flags can optionally be specified in a configuration file, along with additional options.
 
-> *Note: All package.json options are scoped under the `"snowpack"` property.*
+#### Default config
 
-* `"webDependencies"`: (Recommended) Set exactly which packages to install with Snowpack. Without this, Snowpack will just try to install every package in your "dependencies" config. That behavior is great for getting started but it won't warn you if an expected package fails to install.
-* `"namedExports"`: (Optional) If needed, you can explicitly define named exports for any dependency. You should only use this if you're getting `"'X' is not exported by Y"` errors without it. See [rollup-plugin-commonjs](https://github.com/rollup/rollup-plugin-commonjs#usage) for more info.
-* `"dedupe"`: (Optional) If needed, force resolving for these modules to root's node_modules. This helps prevent bundling package multiple time if package is imported from dependencies. See [rollup-plugin-node-resolve](https://github.com/rollup/plugins/tree/master/packages/node-resolve#usage). This is useful when developing a dependency locally, and prevent rollup to duplicate dependencies included both in local and remote packages.
+Snowpack can be configured within `package.json` under the `snowpack` namespace:
 
-```js
+```json
```

Comment:
the reason I use js here instead of JSON is that comments aren't allowed in JSON.

Reply:
Oh right! Will revert.